### PR TITLE
javascript/annalyns-infiltration: add analyzer comments

### PR DESCRIPTION
**Pull request by SleeplessByte**

### CODE REVIEW

**Review by neenjaw — APPROVED** [review]

**Comment by neenjaw** [comment]
LGTM

File: analyzer-comments/javascript/annalyns-infiltration/can_simplify_conditionals.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Simplify or remove if statements.
+
+This exercise is meant to explore booleans and boolean operators.
```
Comment (kotp) [inline analyzer-comments/javascript/annalyns-infiltration/can_simplify_conditionals.md:3]: The word "Boolean" is capitalized, even the British National Corpus agrees.

**Comment by junedev** [comment]
@SleeplessByte Once this is merged, can you report back in Slack whether the hacktoberfest-accepted label worked? From the latest version of the "Rules" section I am not sure anymore it works without also setting the topic for the repo.

**Review by SaschaMann — APPROVED** [review]

File: analyzer-comments/javascript/annalyns-infiltration/can_simplify_conditionals.md (hunk 1 above)
Comment (SleeplessByte) [reply analyzer-comments/javascript/annalyns-infiltration/can_simplify_conditionals.md:3]: I agree too, but in JS documentation, `Boolean` is the function and `boolean` is the type (see: [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean))

**Pull request merged**